From hjk@linutronix.de  Mon Jun 14 15:26:07 2010
From: "Hans J. Koch" <hjk@linutronix.de>
Date: Thu, 10 Jun 2010 01:12:32 +0200
Subject: uio: Remove IRQF_DISABLED flag from uio_pdrv_genirq.c
Cc: Magnus Damm <magnus.damm@gmail.com>, Greg KH <gregkh@suse.de>
Message-ID: <20100609231232.GC2555@local>
Content-Disposition: inline


Remove IRQF_DISABLED flag since it is deprecated and a no-op in the
current kernel.

Signed-off-by: Hans J. Koch <hjk@linutronix.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/uio/uio_pdrv_genirq.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/uio/uio_pdrv_genirq.c
+++ b/drivers/uio/uio_pdrv_genirq.c
@@ -155,7 +155,6 @@ static int uio_pdrv_genirq_probe(struct
 	 * Interrupt sharing is not supported.
 	 */
 
-	uioinfo->irq_flags |= IRQF_DISABLED;
 	uioinfo->handler = uio_pdrv_genirq_handler;
 	uioinfo->irqcontrol = uio_pdrv_genirq_irqcontrol;
 	uioinfo->open = uio_pdrv_genirq_open;
